internal/controller/controllers/preprovisioningimage_controller.go

```diff
@@ -514,6 +508,11 @@ func (r *PreprovisioningImageReconciler) getIronicConfig(ctx context.Context, lo
 		}
 	}
 
+	userOverride := r.getIronicConfigFromUserOverride(log, infraEnv)
```

Comment:
While we're doing this, can we change and rename this function to just return the image? We don't have user overrides for the URLs.